### PR TITLE
Allow user to specify TTL via "--ttl" option when using "st2 key set" command

### LINKED ISSUE #2343: st2 key set in CLI doesn't allow one to set a TTL.
Reported by: psychopenguin on slack

### PR DESCRIPTION
**Pull request by Kami**
This pull request allows users to specify datastore item expiration (TTL) via ``--ttl`` option when using ``st2 key set`` command (e.g. ``st2 key set key1 value1 --ttl=60``).

It also updates ``key get`` and ``key set`` command to display ``expire_timestamp`` attribute.

This resolves #2343

### CODE REVIEW

File: st2client/st2client/commands/keyvalue.py [hunk 1]
```diff
@@ -99,6 +99,8 @@ def __init__(self, resource, *args, **kwargs):
                                  metavar='name',
                                  help='Name of the key value pair.')
         self.parser.add_argument('value', help='Value paired with the key.')
+        self.parser.add_argument('--ttl', type=int, default=None,
+                                 help='TTL for this value.')
```
Comment (manasdk) [inline st2client/st2client/commands/keyvalue.py]: Can you specify unit of TTL in help? I assume it is seconds

**Comment by manasdk** [comment]
+1

File: st2client/st2client/commands/keyvalue.py (hunk 1 above)
Comment (lakshmi-kannan) [inline st2client/st2client/commands/keyvalue.py]: Something to think about is the consistency with st2 token create --ttl. I think that supports `-l`. Please double check and make them consistent.

**Comment by lakshmi-kannan** [comment]
Definitely needs documentation. LGTM otherwise.

File: st2client/st2client/commands/keyvalue.py (hunk 1 above)
Comment (Kami) [inline st2client/st2client/commands/keyvalue.py]: > Can you specify unit of TTL in help? I assume it is seconds

Sure thing.

Comment (Kami) [inline st2client/st2client/commands/keyvalue.py]: > Something to think about is the consistency with st2 token create --ttl. I think that supports -l. Please double check and make them consistent.

Added `-l` ttl option "alias" for consistency.

**Pull request merged**